src/main/resources/run-docker-container.sh

```diff
@@ -3,6 +3,7 @@
 exec java \
     -DLOG_LEVEL=${LOG_LEVEL} \
     -Ds3proxy.endpoint=http://0.0.0.0:80 \
+    -Ds3proxy.virtual-host=${S3PROXY_VIRTUALHOST} \
```

Comment:
Perhaps we just need to have the configuration methods consider `null` and the empty string to be the same thing.